src/pages/AddServers/DeployAddServers.js

```diff
@@ -0,0 +1,253 @@
+// (c) Copyright 2018 SUSE LLC
+/**
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+**/
+
+import React from 'react';
+import BaseUpdateWizardPage from '../BaseUpdateWizardPage.js';
+import { LoadingMask } from '../../components/LoadingMask.js';
+import { ErrorBanner, WarningMessage } from '../../components/Messages.js';
+import { PlaybookProgress } from '../../components/PlaybookProcess.js';
+import { translate } from '../../localization/localize.js';
+import {
+  STATUS, WIPE_DISKS_PLAYBOOK, ARDANA_GEN_HOSTS_FILE_PLAYBOOK,
+  SITE_PLAYBOOK, MONASCA_DEPLOY_PLAYBOOK
+} from '../../utils/constants.js';
+import { fetchJson } from '../../utils/RestUtils.js';
+
+
```

Comment:
I think its fine to have these commented out ansible commands here, but it would be nice to have a one liner or two explaining what they are

Reply:
k , will remove them